src/olympia/browse/templates/browse/macros.html

```diff
@@ -3,21 +3,21 @@
   <div class="other-categories highlight">
     <h3><a href="{{ base_url }}">{{ title }}</a></h3>
     {% if filter %}
-      <ul>
```

Comment:
Hey @atiqueahmedziad, can you fix the indentation here? Like @muffinresearch said, it should be the way it already was.